solr/contrib/ltr/src/java/org/apache/solr/ltr/rest/ManagedModelStore.java

```diff
@@ -189,10 +189,6 @@ public LTRScoringAlgorithm makeLTRScoringAlgorithm(Map<String,Object> map)
             (Map<String,Object>) modelFeature;
         final Feature feature = parseFeature(modelFeatureMap,
             fstore);
-        if (!fstore.containsFeature(feature.getName())) {
-          throw new ModelException("missing feature " + feature.getName()
```

Comment:
This is a valid model check.  If you deploy a model using features that don't exist, you should get an error